### PR TITLE
Navigation Template UAE

### PR DESCRIPTION
**Pull request by LucasIvankio**
Code with the template for the navigation task.

### CODE REVIEW

File: Cpp/src/main/include/Constants.h [hunk 1]
```diff
@@ -22,16 +22,18 @@ namespace constant
...
     /**
      * Digital I/O
      */
+
     static constexpr int START_BUTTON       = 9;  // Digital Input 9
-    static constexpr int E_STOP_BUTTON      = 11; // Digital Input 11
+    static constexpr int E_STOP_BUTTON      = 8; // Digital Input 11
     static constexpr int RUNNING_LED        = 20; // Digital Output 20
     static constexpr int STOPPED_LED        = 21; // Digital Output 21
 }
```
Comment (Jm-Taylor) [inline Cpp/src/main/include/Constants.h:39]: I have not updated the main template yet, but based on the wiring of the robot in Lyon here is the final Constants

- Front Left Motor = 3;
- Front Right Motor = 2;
- Back Left Motor = 1;
- Back Right Motor = 0; 
- Start Button = 11;
- E Stop = 10;
- Running LED = 13;
- Stopped LED = 14;

**Review by Jm-Taylor — CHANGES_REQUESTED** [review]
Please move all this to a separate folder.

Ex. 

- Cpp
- Java
- UAE-Cpp

There are many changes here that could break other competitors' code. Adding the project to its own folder will solve this issue.

Also, please update your Constants.h to reflect the robot wiring.

**Comment by LucasIvankio** [comment]
Hey James,
I've edited the Constants.h to meet the the Robot HW as per orientation.
Also, Ive moved the code to a separated folder.

File: Cpp/src/main/include/Constants.h (hunk 1 above)
Comment (LucasIvankio) [reply Cpp/src/main/include/Constants.h:39]: Done

**Review by Jm-Taylor — APPROVED** [review]

**Pull request merged**